CsCore/CsCoreUnity/Plugins/CsCoreUnity/com/csutil/logging/PersistedImageRegression.cs

```diff
@@ -156,7 +156,7 @@ public class Config {
         }
 
 #if ENABLE_IMAGE_MAGICK
-        private bool AreVisuallyDifferent(Texture2D img1, Texture2D img2, Config config) {
+        private bool AreVisuallyDifferent(Texture2D img1, Texture2D img2, ConFfig config) {
```

Comment:
`ConFfig` looks like a typo? and is also not used anywhere else, so probably also a compile error, could you comment on this @XANGEnino

Reply:
This is a typo. I must have accidentally put an "F" there without realising it.